src/coprocessor/dag/expr/builtin_encryption.rs

```diff
@@ -175,6 +209,122 @@ impl ScalarFunc {
         }
         Ok(Some(i64::from(LittleEndian::read_u32(&input[0..4]))))
     }
+
+    pub fn aes_encrypt<'a, 'b: 'a>(
+        &'b self,
+        ctx: &mut EvalContext,
+        row: &[Datum],
+    ) -> Result<Option<Cow<'a, [u8]>>> {
+        let crypt = try_opt!(self.children[0].eval_string(ctx, row));
+        let key = try_opt!(self.children[1].eval_string(ctx, row));
+        if ctx.aes_mode.unwrap().iv_required && self.children.len() == 3 {
+            ctx.warnings
+                .append_warning(Error::wrong_args(&"IV error".to_string()));
+            return Ok(None);
+        }
+        let key_size = ctx.aes_mode.unwrap().key_size;
```

Comment:
`unwrap()` is not ideal because it will crash TiKV. Thinking that your browser is crashed just because of web page throws a JavaScript error.

Reply:
It has been modified